test/vdiff/wtr-vdiff.config.js

```diff
@@ -0,0 +1,20 @@
+import { argv } from 'node:process';
+import { createConfig } from '../../src/server/wtr-config.js';
+
+const pattern = type => `test/vdiff/*.${type}.js`;
+
+function getGoldenFlag() {
```

Comment:
I don't think this will be useful outside this repo, where I need to be able to generate goldens that are different than when a test is running. So I put it in here